### PR TITLE
set DPI using mupdf attributes

### PR DESCRIPTION
**Pull request by JorjMcKie**
Addresses 3414

### CODE REVIEW

File: src/__init__.py [hunk 1]
```diff
@@ -732,10 +732,11 @@ def get_pixmap(self, matrix=None, dpi=None, colorspace=None, alpha=0):
         if not cs:
             cs = mupdf.fz_device_rgb()
 
-        pix = mupdf.pdf_new_pixmap_from_annot( self.this, ctm, cs, mupdf.FzSeparations(0), alpha)
+        pix = mupdf.pdf_new_pixmap_from_annot(self.this, ctm, cs, mupdf.FzSeparations(0), alpha)
         if dpi:
-            pix.set_dpi(dpi, dpi)
-        return Pixmap( pix)
+            pix.m_internal.xres = dpi
+            pix.m_internal.yres = dpi
```
Comment (julian-smith-artifex-com) [inline src/__init__.py:738]: Why this change - isn't it just duplicating the code in `Pixmap.set_dpi()`?

Comment (julian-smith-artifex-com) [reply src/__init__.py:738]: Oh, i see the reference to #3414. But might it be better to do:
```
ret = Pixmap(pix)
ret.set_dpi(dpi, dpi)
return ret
```
? This way we make use of the existing `Pixmap.set_dpi()`, instead of duplicating it.

Comment (JorjMcKie) [reply src/__init__.py:738]: Ah, ok - didn't know this would work like that.

Comment (JorjMcKie) [reply src/__init__.py]: fixed

**Review by julian-smith-artifex-com — APPROVED** [review]
